frontend/src/components/widgets/DataGrid/DataGrid.test.tsx

```diff
@@ -82,6 +83,50 @@ describe("DataGrid widget", () => {
     expect((result.current.columns[0] as SizedGridColumn).width).toBe(321)
   })
 
-  // TODO(lukasmasuch): Unit tests for a few methods, such as fillCellTemplate, getColumn, useDataLoader, getCellTemplate, etc.
-  //                    will be added in a later PR once support for different data types is added.
+  it("should correctly sort the table descending order", () => {
+    // TODO(lukasmasuch): Add additional sort tests for other example quiver tables
+    const tableColumns = getColumns(new Quiver({ data: TEN_BY_TEN }))
+
+    // Add descending sort for first column
+    const { result } = renderHook(() =>
+      useDataLoader(new Quiver({ data: TEN_BY_TEN }), {
+        column: tableColumns[0],
+        mode: "smart",
+        direction: "desc",
+      })
+    )
+
+    const sortedData = []
+
+    for (let i = 0; i < result.current.numRows; i++) {
+      sortedData.push(
+        (result.current.getCellContent([0, i]) as NumberCell).data
+      )
+    }
+
+    expect(sortedData).toBe(sortedData.sort().reverse())
```

Comment:
I don't think this test is actually testing anything as written. The reason is that `.sort()` and `.reverse()` both mutate the array that they are called on as well as return it, so this expect is essentially

```javascript
expect(sortedData).toBe(sortedData)
```

I was originally tipped off that this might be the case because I think `toBe` tests for address equality (vs `toEqual`, which tests for structural equality), so I was surprised to see these tests passing and double checked the behavior of `.sort()` and `.reverse()`.

Same comment for the test below

Reply:
Good catch :) Fixed it by using `toEqual()` and by creating new copies of the array.